### ISSUE TITLE
Call wrapped method from callsFake

### ISSUE THREAD

**Issue by servel333** [sinonjs/sinon#1928]
**Is your feature request related to a problem? Please describe.**

I want to call the original method from inside `callsFake`. This is different than using `callThrough` because there are no conditions to skip, or the conditions are written in JS inside the `callsFake` method rather than Sinon syntax. I could use `wrappedMethod` but it is not documented, so I'm worried it is internal and may change.

**Describe the solution you'd like**

```
sinon.stub(obj, 'method').callsFake((...args) => {
  if (condition) {
    // Different behavior
    return;
  }

  this.callWrapped();
  obj.method.callWrapped();
  this.callWrappedWithArgs(...args);
  obj.method.callWrappedWithArgs(...args);
});
```

OR

Add `wrappedMethod` to the documentation so it doesn't change.

**Describe alternatives you've considered**

```
sinon.stub(obj, 'method').callsFake((...args) => {
  if (condition) {
    // Different behavior
    return;
  }

  // Not documented
  obj.method.wrappedMethod.apply(this, args);
});
```

**Additional context**

**Comment by fatso83** [comment]
I think this needs to be documented. I recall us discussing this before.

**Comment by sergio-domingues** [comment]
Any updates on this?

**Comment by fatso83** [comment]
https://sinonjs.org/releases/latest/stubs/#stubwrappedmethod

Closed in 1e81ec32d
